gmm/handler.go

```diff
@@ -583,6 +585,9 @@ func HandleInitialRegistration(ue *context.AmfUe, anType models.AccessType) erro
 
 	if len(ue.AllowedNssai[anType]) == 0 {
 		gmm_message.SendRegistrationReject(ue.RanUe[anType], nasMessage.Cause5GMM5GSServicesNotAllowed, "")
+		ngap_message.SendUEContextReleaseCommand(ue.RanUe[anType], context.UeContextN2NormalRelease,
+			ngapType.CausePresentNas, ngapType.CauseNasPresentNormalRelease)
+		ue.Remove()
```

Comment:
Can you create call flow for this problem and fix ?

Reply:
@thakurajayL, it looks like this comment is outdated. Correct?

Reply:
I would prefer to take code as is. Valid fix to cleanup N1 tunnel.

Reply:
@thakurajayL, so, should we approve this PR?